LICENSE

```diff
@@ -1,6 +1,7 @@
 (The MIT License)
 
-Copyright (c) 2011-2018 JS Foundation and contributors, https://js.foundation
+Copyright (c) 2011-2019 OpenJS Foundation and contributors, https://openjsf.org/
+
```

Comment:
I don't think this is correct. This is legal, therefore quite sensitive stuff.
Is it still MIT license?
2011-2019: OpenJS Foundation exists since 2018?
This concerns legal issues and you have to be careful what you are changing.

Reply:
@juergba I didn't know a legal issue. 
How we descibe the copyright? "Since 2011" part needed?

`Copyright (c) 2011-2019 OpenJS Foundation`

Reply:
@juergba I didn't know a legal issue. 
How we descibe the copyright? "Since 2011" part needed?

`Copyright (c) 2011-2019 OpenJS Foundation`

Reply:
I don't know. You could study OpenJSFoundation's homepage, or study repos of other packages, or ask jorydotcom or ask boneskull.

Reply:
@ jorydotcom 
Do you know about this legal issue? 
`Copyright (c) 2011-2019 OpenJS Foundation` or `Copyright (c) 2019 OpenJS Foundation`. 
We don't know a legal issue.

Reply:
@jorydotcom
Do you know about this legal issue? 
`Copyright (c) 2011-2019 OpenJS Foundation` or `Copyright (c) 2019 OpenJS Foundation`. 
We don't know a legal issue.